samples/sample_common/include/sample_defs.h

```diff
@@ -116,7 +116,7 @@ enum LibVABackend
 #define MSDK_ENC_WAIT_INTERVAL 300000
 #define MSDK_VPP_WAIT_INTERVAL 300000
 #define MSDK_SURFACE_WAIT_INTERVAL 300000
-#define MSDK_DEVICE_FREE_WAIT_INTERVAL 30000
+#define MSDK_DEVICE_FREE_WAIT_INTERVAL 300000
```

Comment:
And now the question, why do we really need separate MSDK_DEVICE_FREE_WAIT_INTERVAL while generally MFX_WNR_DEVICE_BUSY can last the same time as synchronization of some internal MSDK task? In other words, it's a time of SyncOperation execution. And for SyncOperation we use MSDK_WAIT_INTERVAL.